chicago/bills.py

```diff
@@ -97,11 +96,15 @@ def scrape(self):
                                                            'communication', 
                                                            'report', 
                                                            'oath of office') :
-                    continue
+                    bill_type = None
                 else :
                     bill_type = legislation_summary['Type'].lower()
 
-                bill_session = self.session(legislation_summary['Intro\xa0Date'])
+                try :
+                    bill_session = self.session(self.toTime(legislation_summary['Intro\xa0Date']))
+                except :
+                    print(legislation_summary)
```

Comment:
Would it make more sense to get legislation_summary into an error message instead of printing so it's clearer what we're looking at?